gatsby-config.js

```diff
@@ -19,6 +20,7 @@ const skipWebpackAnalyzer = has(process.env, 'GITHUB_ACTIONS') || has(process.en
 
 const getSentryEnvironment = () => {
   if (get(process.env, 'NODE_ENV') === 'production') return 'production';
+  if (get(process.env, 'NODE_ENV') === 'test') return 'development';
```

Comment:
We don't have a `test` environment in sentry and we don't really need one. `development` will do.